build.gradle

```diff
@@ -200,9 +201,9 @@ sonarqube {
     property "sonar.pitest.mode", "reuseReport"
     property "sonar.pitest.reportsDirectory", "build/reports/pitest"
     property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/config/**," +
-                                 "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/Application.java" +
-                                 "src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/**," +
-                                 "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/clients/**"
+        "src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/Application.java" +
```

Comment:
seems like different people have different format settings?

Reply:
looks like my groovy settings have changed from 2 spaces to 4 spaces on continuation i will change to 2